### PR TITLE
docs(readme): add related project: lighthouse-viewer

### PR DESCRIPTION
**Pull request by dvelasquez**
**Summary**

This PR updates the README.md adding 4 small projects I'm maintaining.
* lighthouse-viewer: lighthouse-reporter sourcecode packaged as an UMD & ESM npm package, to visualise your JSON reports. The process to copy the code from lighthouse to this package is automated.
* react2-lighthouse-viewer: This package is a React wrapper of `lighthouse-viewer`.
* svelte-lighthouse-viewer: This package is a Svelte wrapper of `lighthouse-viewer`.
* vue-lighthouse-viewer: This package is a Vue wrapper of `lighthouse-viewer`.

Add more options for visualisation of lighthouse JSON reports.

Sourcecode of the projects added: https://github.com/dvelasquez/lighthouse-viewer
**Related Issues/PRs**

**Summary**

This change just add 4 new projects to the **related projects** section.

* Repository: https://github.com/dvelasquez/lighthouse-viewer
* CodeSandbox Demo: https://codesandbox.io/s/lighthouse-viewer-vanillajs-demo-zw2bj
* Demo/Documentation: https://dvelasquez.github.io/lighthouse-viewer/index.html
**Related Issues/PRs**

### CODE REVIEW

**Comment by dvelasquez** [comment]
@googlebot I signed it!

**Comment by dvelasquez** [comment]
@brendankenny Yes, is one repository with 4 npm packages. That's why I left them separate.
I could change the links to NPM instead where is more clear that are separate packages instead of github, where is a Monorepo/multipackage.

**Comment by connorjclark** [comment]
This is neat, thanks for making and sharing!

> Yes, is one repository with 4 npm packages. That's why I left them separate.
I could change the links to NPM instead where is more clear that are separate packages instead of github, where is a Monorepo/multipackage.

They're still all fundamentally the same thing: embedding the report viewer in vanilla JS or some given framework. Can you keep just the link to `https://github.com/dvelasquez/lighthouse-viewer`, with language in the description that there is support for react, vue, etc... ?
____

I noticed some issues, so here's some unsolicited issue reporting. :)

image
URL: https://dvelasquez.github.io/lighthouse-viewer/packages/react2-lighthouse-viewer/demo/index.html
(happens when you click on a big category gauge)

image
URL: https://codesandbox.io/s/lighthouse-viewer-vanillajs-demo-zw2bj

File: readme.md [hunk 1]
```diff
@@ -383,10 +383,14 @@ Other awesome open source projects that use Lighthouse.
 * **[lighthouse-mocha-example](https://github.com/rishichawda/lighthouse-mocha-example)** - Run Lighthouse performance tests with Mocha and chrome-launcher.
 * **[lighthouse-monitor](https://github.com/verivox/lighthouse-monitor)** - Run Lighthouse against all your URLs. Send metrics to any backend you want, save all reports with automatic data retention, and compare any two results in a web UI.
 * **[lighthouse-persist](https://github.com/foo-software/lighthouse-persist)** - Run Lighthouse and upload HTML reports to an AWS S3 bucket.
+* **[lighthouse-viewer](https://github.com/dvelasquez/lighthouse-viewer/tree/main/packages/lighthouse-viewer)** - Render a Lighthouse JSON report with Vanilla JS. This is just the original Lighthouse Reporter sourcecode packaged as UMD and ESM.
```
Comment (paulirish) [inline readme.md:386]: how about this?

```suggestion
* **[lighthouse-viewer](https://github.com/dvelasquez/lighthouse-viewer/tree/main/packages/lighthouse-viewer)** - Render the Lighthouse JSON into a report, using the Lighthouse Report Renderer repackaged as UMD and ESM. Also available with React, Svelte and Vue wrappers.
```

and given that we can drop the 3 extra ones.

Comment (dvelasquez) [reply readme.md:386]: Like your proposal, will take it! Thanks @paulirish

**Comment by dvelasquez** [comment]
> This is neat, thanks for making and sharing!
> 
> > Yes, is one repository with 4 npm packages. That's why I left them separate.
> > I could change the links to NPM instead where is more clear that are separate packages instead of github, where is a Monorepo/multipackage.
> 
> They're still all fundamentally the same thing: embedding the report viewer in vanilla JS or some given framework. Can you keep just the link to `https://github.com/dvelasquez/lighthouse-viewer`, with language in the description that there is support for react, vue, etc... ?
> 
> I noticed some issues, so here's some unsolicited issue reporting. :)
> 
> image
> URL: https://dvelasquez.github.io/lighthouse-viewer/packages/react2-lighthouse-viewer/demo/index.html
> (happens when you click on a big category gauge. can't scroll back up)
> 
> image
> URL: https://codesandbox.io/s/lighthouse-viewer-vanillajs-demo-zw2bj

I will take a look on this. One of the problems I got was with the use of MVP.css, a classless css framework, but because of this, several of the elements of the report itself were override. That's why I'm loading the report inside an iFrame.
So I will drop this framework and use a different one for the demos.
And maybe will make some sense to make some changes in the report itself to avoid involuntary overrides of styles.

**Pull request merged**